src/modules/tests/tests.model.ts

```diff
@@ -1,12 +1,15 @@
 import { StandardCarTestCATBSchema } from '@dvsa/mes-test-schema/categories/B';
 import { TestStatus } from './test-status/test-status.model';
+import { StandardTrailerTestCATBESchema } from '@dvsa/mes-test-schema/categories/BE';
+
+export type TestResultUnionType = StandardCarTestCATBSchema | StandardTrailerTestCATBESchema;
```

Comment:
can be removed if we use the one in api definitions

Reply:
I think we'll merge this work in as-is and then do the work to implement the new real schemas later